charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -12,7 +12,11 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
 data:
+  {{ if .Values.airflowReleasesConfig }}
+  airflow_releases.json:
```

Comment:
@andriisoldatenko should we change this to yaml extension now?

Reply:
@schnie i fixed code, PTAL

Reply:
I'm trying to respect original file from https://github.com/astronomer/houston-api/blob/main/airflow_releases.json,
so we define `airflowReleasesConfig` and than convert to json via `toJson`
```
data:
  {{ if .Values.houston.airflowReleasesConfig }}
  airflow_releases.json: |
    {{ .Values.houston.airflowReleasesConfig | toJson }}
  # These are system-specified config overrides.
  {{- end }}
```
and inside pod we can see correct file body and correct extension.
@schnie please correct me if i'm wrong.

Reply:
```$ kubectl exec -it astronomer-houston-5f7f765f4b-qzvdb -n astronomer sh
/houston $
/houston $ cat airflow_releases.json
{"available_releases":[{"channel":"stable","level":"new_feature","release_date":"2020-09-29T22:15:00+00:00","tags":["2.0.0-alpine3.10-onbuild"],"url":"https://github.com/astronomer/airflow/releases/tag/v1.10.12%2Bastro.1","version":"2.0.0"}]}
/houston $
```